website/content/docs/discovery/dns.mdx

```diff
@@ -335,37 +350,22 @@ using the [`advertise-wan`](/docs/agent/options#_advertise-wan) and
 [`translate_wan_addrs`](/docs/agent/options#translate_wan_addrs) configuration
 options.
 
-## Namespaced Services <EnterpriseAlert inline />
-
-Consul Enterprise 1.7.0 added support for namespaces including resolving namespaced
-services via DNS. To maintain backwards compatibility existing queries can be used
-and these will resolve services within the `default` namespace. However, for resolving
-services from other namespaces the following form can be used:
-
-```text
-[tag.]<service>.service.<namespace>.<datacenter>.<domain>
-```
-
-This is the canonical name of a Consul Enterprise service with all parts present. Like
-Consul OSS some parts may be omitted but which parts depend on the value of the
-[`prefer_namespace` configuration](/docs/agent/options#dns_prefer_namespace).
-
-With `prefer_namespace` set to `true` the datacenter may be omitted and will be defaulted
-to the local agents datacenter:
-
-```text
-[tag.]<service>.service.<namespace>.<domain>
-```
+## Namespaced/Partitioned Services <EnterpriseAlert inline />
 
-With `prefer_namespace` set to `false` the namespace may be omitted and will be defaulted
-to the `default` namespace:
+Consul Enterprise supports resolving namespaced and partitioned services via DNS.
+To maintain backwards compatibility existing queries can be used and these will
+resolve services within the `default` namespace and partition. However, for resolving
+services from other namespaces or partitions the following form can be used:
 
 ```text
-[tag.]<service>.service.<datacenter>
+[tag.]<service>.service[.<namespace>.ns][.<partition>.ap][.<datacenter>.dc].<domain>
```

Comment:
Should the brackets be removed from the later parts, since they're not optional the way tags are?